### ISSUE TITLE
Site language selector

### ISSUE THREAD

**Issue by vincentbesanceney** [jbms/sphinx-immaterial#297]
From what I see in `src/partials/header.html`, the theme already supports this feature, but it doesn't seem to be configurable from theme options. I tested a change to the theme to make language selection available from customization, adding `alternate` languages just [as it is done in the mkdocs material theme](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language-selector). Is this the way to go or do you favor another option?

**Comment by 2bndy5** [comment]
I suppose we could add support for this.

If you're using ReadTheDocs, then the language can be selected via the RTD floating dropr-down menu: https://docs.readthedocs.io/en/stable/localization.html

**Comment by 2bndy5** [comment]
I'm not particularly happy with the config `html_theme_options['extra']['alternative']` as it isn't an obvious name for such an option. I would probably add a key named `languages` that takes a map of the same options used upstream:
```py
html_theme_options = {
    "languages": [
        {
            "name": "English",
            "lang": "en",
            "link": "en/",
        },
        {
            "name": "Frebch",
            "lang": "fr",
            "link": "fr/",
        },
    ]
}

**Comment by 2bndy5** [comment]
Customizing the icon should probably get a different key as well.

### In upstream config
```yml
theme:
  icon:
    alternate: material/translate
```
### In our config
```py
html_theme_options: {
    "icon": {
        "languages": "material/translate",
    }
}
```

**Comment by 2bndy5** [comment]
On second thought, It would be easier (for development) to customize the icon using the same key used upstream.
```py
html_theme_options = {
    "icon": {
        "alternate": "material/translate",
    }
}
```

**Comment by 2bndy5** [comment]
I started a branch called `lang-drop-down` that has the added config to support this, but it needs documentation.
Initial attempt worked with the following config:
```py
html_theme_options = {
    "languages": [
        {
            "name": "English",
            "lang": "en",
            "link": "en/",
        },
        {
            "name": "Frebch",
            "lang": "fr",
            "link": "fr/",
        },
    ],
    "icon": {
        "alternate": "fontawesome/solid/language",
    }
}
```

**Comment by 2bndy5** [comment]
Feature available in [v0.11.8](https://pypi.org/project/sphinx-immaterial/0.11.8/).
